### PR TITLE
CI: trigger PL for lite

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

As PL is not dependent on Lite, it makes sense to run testing of PL when ever we make changes in Lite...

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @carmocca @akihironitta @borda @justusschock @awaelchli

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

File: .github/workflows/ci-pytorch-tests.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@ on:
     branches: [master, "release/*"]
     types: [opened, reopened, ready_for_review, synchronize]  # add `ready_for_review` since draft is skipped
     paths:
+      - "requirements/lite/**"
```
Comment (carmocca) [inline .github/workflows/ci-pytorch-tests.yml:11]: These are duplicated

File: .github/workflows/ci-pytorch-tests-slow.yml [hunk 2]
```diff
@@ -9,6 +9,8 @@ on:
     types: [opened, reopened, ready_for_review, synchronize]  # add `ready_for_review` since draft is skipped
     paths:
       - ".github/workflows/ci-pytorch-tests-slow.yml"
+      - "requirements/lite/**"
```
Comment (carmocca) [inline .github/workflows/ci-pytorch-tests-slow.yml:12]: These are duplicated

**Review by carmocca — CHANGES_REQUESTED** [review]
Can you add the new accelerator jobs to checkgroup?

File: .github/workflows/ci-pytorch-tests-slow.yml [hunk 3]
```diff
@@ -9,6 +9,8 @@ on:
...
+      - "src/lightning_lite/**"
```
Comment (Borda) [inline .github/workflows/ci-pytorch-tests-slow.yml:13]:
```suggestion
```

File: .github/workflows/ci-pytorch-tests.yml [hunk 4]
```diff
@@ -8,6 +8,8 @@ on:
...
+      - "src/lightning_lite/**"
```
Comment (Borda) [inline .github/workflows/ci-pytorch-tests.yml:12]:
```suggestion
```

File: .github/workflows/ci-pytorch-tests.yml (hunk 1 above)
Comment (Borda) [reply .github/workflows/ci-pytorch-tests.yml]: good catch, fixed :)

File: .github/workflows/ci-pytorch-tests-slow.yml (hunk 2 above)
Comment (Borda) [reply .github/workflows/ci-pytorch-tests-slow.yml]: good catch, fixed :)

**Comment by Borda** [comment]
> Can you add the new accelerator jobs to checkgroup?

I think it is fine, but if you suggest, I can do it :)

**Review by carmocca — APPROVED** [review]

**Comment by carmocca** [comment]
Done!

Note that this is a bit overkill because Lite does not support HPUs or IPUs, so there's very little code shared. But it's okay since tests run fast.

**Pull request merged**